nav2_gps_waypoint_follower_demo/models/dumpster/materials/scripts/dumpster.material

```diff
@@ -0,0 +1,29 @@
+material Dumpster/Diffuse
```

Comment:
Why are you uploading all of these files? Are these not provided by gazebo that you were just using? They'll be downloaded on use from gazebo index, you don't need to upload these here unless you created them or had to manually download them youself

Reply:
some of them are provided in `gazebo_models` but some are not, I think to ensure that gazebo world will properly launch its just better to provide all models used in the world file.

Reply:
The ones that are, can we not include them here? I don't think there's a need to duplicate everything, that's alot of "stuff" to clone. I thought by including a `model://` it would look in the model store and download new things if needed?

Reply:
As I know if Gazebo could not find a model it will then run to get models from [gazebo_models](https://github.com/osrf/gazebo_models.git) and download them to `~/.gazebo/models`.